docs/updating-dependencies.md

```diff
@@ -9,37 +12,23 @@ To update golang dependencies one has to choose between two options:
 ## Update the whole dependency tree
 
 Done when the need to catch upstream dependencies arises, it can be done by
-calling
-
-```bash
-make vendor
-```
-
-in root folder, which calls `hack/update-go-module-dependencies.sh`.
-
-The reason for calling script instead of directly calling:
+calling:
 
 ```bash
+# Ensure go module-aware mode is set to auto
+export GO111MODULE=auto
```

Comment:
My work laptop is fubar so I can't test this myself, but I am 99% sure this breaks re: mac vs linux. Can we get an SRE from each side of the aisle to attempt this change?

Reply:
Woops, ok, so maybe it was a Linux thing - have my laptop back up and I, for reasons unknown along my dev journey on this team, have changed my `GO111MODULE=""`, I can test this now, so I'll give it a go and see what happens :)